perfkitbenchmarker/benchmarks/mongodb_ycsb_benchmark.py

```diff
@@ -56,22 +76,11 @@ def Prepare(benchmark_spec):
   """
   vms = benchmark_spec.vms
   assert len(vms) == BENCHMARK_INFO['num_machines']
-  mongo_vm = vms[0]
-
-  # Install mongodb on the 1st machine.
-  mongo_vm.Install('mongodb_server')
-  data_dir = _GetDataDir(mongo_vm)
-  mongo_vm.RemoteCommand('mkdir {0} && chmod a+rwx {0}'.format(data_dir))
-  mongo_vm.RemoteCommand(
-      "sudo sed -i -e '/bind_ip/ s/^/#/; s,^dbpath=.*,dbpath=%s,' %s" %
-      (data_dir,
-       mongo_vm.GetPathToConfig('mongodb_server')))
-  mongo_vm.RemoteCommand('sudo service %s restart' %
-                         mongo_vm.GetServiceName('mongodb_server'))
+  mongo_vm, ycsb_vm = vms[:2]
 
-  # Setup YCSB load generator on the 2nd machine.
-  ycsb_vm = vms[1]
-  ycsb_vm.Install('ycsb')
+  vm_util.RunThreaded(apply,
```

Comment:
apply has been deprecated since python 2.3 and was completely removed in python 3 - maybe just use a lambda?